HucaresReact/src/components/Modal/InfoEditingModal.jsx

```diff
@@ -7,21 +8,67 @@ export class InfoEditingModal extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
+      open: false,
       forms: props.data.formFields,
       checkboxes: props.data.checkboxes,
+      callback: props.callback,
     };
   }
 
+    close = () => {this.setState({open: false})};
+
+    currentDateIfNull(date) {
+      if (date === undefined) {
+        return new Date();
+      } else {
+        return new Date(date);
+      }
+    }
+
+    convertDate(unixDate) {
+      const date = this.currentDateIfNull(unixDate);
+
+      const day = date.getDate();       
+      const month = date.getMonth() + 1;
+      const year = date.getFullYear(); 
+      const hour = date.getHours();
+      const minute = date.getMinutes();
+      const second = date.getSeconds(); 
+
+      return day + "/" + month + "/" + year + " " + hour + ':' + minute + ':' + second; 
+    }
+
+    onSubmitAction() {
+      this.close();
+      axios
+          .post(`${process.env.HUCARES_API_BASE_URL}/api/mlp/insert`,
+              {
+                  headers: { 'Access-Control-Allow-Origin': '*' },
+                  plateNumber: this.state.forms[0].value,
+                  searchStartDatetime: this.convertDate(this.state.forms[1].value)
+              })
+          .then((response) =>
+            {
+              this.forceUpdate();
+              if(this.state.callback !== undefined) this.state.callback();
```

Comment:
I think `if(this.state.callback) this.state.callback();` would be enough, but not sure